### DISCUSSION TITLE
Elastalert email not emailing.

### DISCUSSION THREAD

**Discussion by 2nutz4u** [Security-Onion-Solutions/securityonion#13719] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

293GB

### Storage for /nsm

700G

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

No email alerts from elastalert. 
Steps I have taken:
1.  I followed the documentation https://svr-securityonion/docs/notifications.html 
![msedge_48BHggntbN](https://github.com/user-attachments/assets/35dd074f-5b55-4cf2-b269-ed21058515ba)
![msedge_Cgdpywe9wU](https://github.com/user-attachments/assets/8f016216-00b8-4e5e-9e3e-32dab17e0bd8)
2.  Created a detection rule. 
![msedge_ebZpkNebBb](https://github.com/user-attachments/assets/19d46288-5c8c-47af-b2dd-cb21e23f21fb)
3. I do see the alerts in the dashboard but no email.
![msedge_pr21uZi8Cw](https://github.com/user-attachments/assets/73721e5c-8f67-4679-b49c-40d9b37f84f8)

I have tried with and without email  authentication. 
Did I miss a step?

Please advise. Thank you.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment]
This is a pro feature. You can contact support if you have pro and they can assist in taking a look at this. If you do not own pro you will have to manage Elastalert rules and alerters via the cli by dropping your elastalert rules in `/opt/so/saltstack/local/salt/elastalert/files/custom` 

Please note that this is the exact same way it was done prior to 2.4.70.

**Comment by 2nutz4u** [comment]
Thank you for the clarification. Do you have a sample custom rule or documentation that I can refer to?

**Comment by 2nutz4u** [comment]
Anyone? I have been struggling to get the rule to work.

**Comment by TOoSmOotH** [comment]
https://docs.securityonion.net/en/2.4/elastalert.html

https://elastalert2.readthedocs.io/en/latest/elastalert.html#overview
